datarepo-clienttests/src/main/java/testscripts/IngestFile.java

```diff
@@ -77,23 +73,31 @@ public void userJourney(ApiClient apiClient) throws Exception {
 
     String targetPath = "/testrunner/IngestFile/" + FileUtils.randomizeName("") + ".txt";
 
-    FileLoadModel fileLoadModel =
-        new FileLoadModel()
+    BulkLoadFileModel fileLoadModel =
+        new BulkLoadFileModel()
             .sourcePath(sourceFileURI.toString())
             .description("IngestFile")
             .mimeType("text/plain")
-            .targetPath(targetPath)
-            .profileId(datasetSummaryModel.getDefaultProfileId());
+            .targetPath(targetPath);
+    List<BulkLoadFileModel> bulkLoadFileModelList = new ArrayList<>();
+    bulkLoadFileModelList.add(fileLoadModel);
+    BulkLoadArrayRequestModel fileLoadModelArray =
+        new BulkLoadArrayRequestModel()
+            .profileId(datasetSummaryModel.getDefaultProfileId())
+            .loadArray(bulkLoadFileModelList);
     JobModel ingestFileJobResponse =
-        repositoryApi.ingestFile(datasetSummaryModel.getId(), fileLoadModel);
+        repositoryApi.bulkFileLoadArray(datasetSummaryModel.getId(), fileLoadModelArray);
+
     ingestFileJobResponse = DataRepoUtils.waitForJobToFinish(repositoryApi, ingestFileJobResponse);
-    FileModel fileModel =
-        DataRepoUtils.expectJobSuccess(repositoryApi, ingestFileJobResponse, FileModel.class);
-    logger.debug(
-        "Successfully ingested file: path = {}, id = {}, size = {}",
-        fileModel.getPath(),
-        fileModel.getFileId(),
-        fileModel.getSize());
+    BulkLoadArrayResultModel bulkLoadArrayResultModel =
+        DataRepoUtils.expectJobSuccess(repositoryApi, ingestFileJobResponse, BulkLoadArrayResultModel.class);
+    BulkLoadFileResultModel fileInfo = bulkLoadArrayResultModel.getLoadFileResults().get(0);
+
+    System.out.println(
```

Comment:
Can you keep this using the logger.debug with {} format?

Reply:
shouldn't it be logger.info to match the other ones?

Reply:
Ah up to you. I usually log at debug level if it isn't relevant to the test outcome, but this is very subjective.